pkg/zcache/combined_cache.go

```diff
@@ -64,10 +63,13 @@ func (c *combinedCache) Get(ctx context.Context, key string, data interface{}) e
 		}
 
 		c.logger.Sugar().Debugf("set value found on remote cache in the local cache, key: [%s]", key)
+		ttl, ttlErr := c.remoteCache.TTL(ctx, key)
+		if ttlErr != nil {
+			c.logger.Sugar().Errorf("error getting TTL for key [%s] from remote cache, err: %s", key, ttlErr)
+		}
 
 		// Refresh data TTL on both caches
-		_ = c.localCache.Set(ctx, key, data, c.ttl)
-		_ = c.remoteCache.Set(ctx, key, data, c.ttl)
+		_ = c.localCache.Set(ctx, key, data, ttl)
```

Comment:
if this fails, ttl will be 0, and it won't never removed locally... shouldn't we skip saving it locally on this case?